proto/qdrant/cloud/cluster/backup/v1/backup.proto

```diff
@@ -137,12 +137,27 @@ message ListBackupsRequest {
   // The identifier of the schedule (in GUID format).
   // When this field is set, only backups triggered by the backup schedule are returned.
   optional string backup_schedule_id = 3 [(buf.validate.field).string = {uuid: true}];
+  // Maximum number of items to return.
+  // If not specified, all items are returned.
+  optional uint32 page_size = 10 [(buf.validate.field).uint32.gt = 0];
+  // A page token, received from a previous call.
+  // Provide this to retrieve the subsequent page.
+  // When paginating, all other parameters provided to the request must match
+  // the call that provided the page token.
+  optional string page_token = 11;
 }
 
 // ListBackupsResponse is the response from the ListBackups function
 message ListBackupsResponse {
   // The actual backups in this list.
+  // When pagination is used it contains a single page only, not all items.
   repeated Backup items = 1;
+  // The total number of items available (useful in relation with pagination).
+  // This field is fill out when pagination is used (aka in the request `page_size` was provided).
+  optional uint32 total_count = 10;
```

Comment:
nitpicking: could this be `total_size` to be consistent with `page_size`? Also it's the one suggested by google in their API-158

![Uploading Screenshot 2025-08-25 at 12.41.09.png…]()

Reply:
what about the int32 vs uint32 (signed vs unsigned)?

Reply:
renamed (and made it int32)